workflow/github.md

```diff
@@ -1,2 +1,138 @@
 # Working on GitHub
 
+This document outlines the rules to contribute and to create issues to a project on the organisation [Canonical-websites](https://github.com/canonical-websites/)
+
+## Table of content
+
+* [How can I contribute?](#how-can-i-contribute-)
+  + [Report Bugs](#report-bugs)
+  + [Suggest Enhancements](#suggest-enhancements)
+  + [Work on an issue](#work-on-an-issue)
+* [Pull Requests](#pull-requests)
+  + [First contribution](#first-contribution)
+  + [Styleguides](#styleguides)
+    - [Commits messages](#commits-messages)
+    - [HTML](#html)
+    - [JS](#js)
+    - [Stylesheets](#stylesheets)
+    - [Python](#python)
+    - [Documentation](#documentation)
+  + [Merging process](#merging-process)
+* [Templates](#templates)
+  + [Issues template](#issues-template)
+  + [Pull-request template](#pull-request-template)
+
+## How can I contribute?
+
+In the Canonical web-sites organisation we work with Github. All types of work are tracked as GitHub issues. You will find in this section how you can contribute to this organisation.
+
+### Report Bugs
+
+Well done you have found a bug! Go to the corresponding project, it should be in the [Canonical websites organisation](https://github.com/canonical-websites/) and open an issue. These are the steps you need follow to add an issue:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Add labels:** see [Labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Suggest Enhancements
+
+If you think details could change on our websites you can suggest enhancements by creating a new issue. These are the steps you need follow to suggest enhancements:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Screenshots:** Don’t forget to add screenshots of suggestion designs to explain
+* **Add labels:** see [labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Work on an issue
+You are free to work on an issue if no one seems to work on it. Don’t forget to assign yourself to the issue. Once you finish the work open a [pull-request](#pull-Requests)
+
+## Pull Requests
+
+In this section you will find a description on how to create a pull-request for a Canonical Web-sites project. These are the steps to create a pull-request:
+* Fill the [template](#pull-request-template)
+* Make sure the tests are OK
+* Make sure the [styleguides](#styleguides) are respected
+* Add the [Labels](./labels.md)
+
+### First contribution
+
+If you want to create a pull-request you will need to fork the project you are working on. From your fork, create a branch and work on your issue.
+
+### Styleguides
+
+#### Commits messages
+
+The commit message must describe the feature you want to add to the project.
+
+#### HTML
```

Comment:
I think given that each of these sections is simply a link to another document, we should collapse it down to a single section. Something like:

### Styleguides

When making changes to any of our projects, please follow our styleguides:

- [HTML](../coding/html.md)
- [JavaScript](../coding/js.md)
- [Stylesheets](../coding/stylesheets.md)

along with any community standards. If we haven't written a styleguide for the language you're working in, please feel free to [suggest one](https://github.com/canonical-webteam/practices/blob/master/CONTRIBUTING.md).